test/integration/restrictusers_test.go

```diff
@@ -3,33 +3,25 @@ package integration
 import (
 	"testing"
 
+	kapi "k8s.io/api/core/v1"
 	rbacv1 "k8s.io/api/rbac/v1"
 	kapierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	kapi "k8s.io/kubernetes/pkg/apis/core"
 
+	authorizationv1 "github.com/openshift/api/authorization/v1"
+	authorizationclient "github.com/openshift/client-go/authorization/clientset/versioned/typed/authorization/v1"
 	authorizationapi "github.com/openshift/origin/pkg/authorization/apis/authorization"
-	authorizationclient "github.com/openshift/origin/pkg/authorization/generated/internalclientset"
 	testutil "github.com/openshift/origin/test/util"
 	testserver "github.com/openshift/origin/test/util/server"
 )
 
 func TestRestrictUsers(t *testing.T) {
-	masterConfig, err := testserver.DefaultMasterOptions()
+	masterConfig, clusterAdminKubeConfig, err := testserver.StartTestMaster()
 	if err != nil {
 		t.Fatalf("error creating config: %v", err)
 	}
 	defer testserver.CleanupMasterEtcd(t, masterConfig)
 
-	masterConfig.KubernetesMasterConfig.APIServerArguments["enable-admission-plugins"] = append(
```

Comment:
restore this block.  YOu just broke the test

Reply:
yup doing that now.